tool/tctl/common/resource_command.go

```diff
@@ -1648,3 +1726,31 @@ If you would still like to proceed, re-run the command with both --force and --c
 }
 
 const managedByStaticDeleteMsg = `This resource is managed by static configuration. In order to reset it to defaults, remove relevant configuration from teleport.yaml and restart the servers.`
+
+func findDeviceByIDOrTag(ctx context.Context, remote devicepb.DeviceTrustServiceClient, idOrTag string, allowMultiple bool) ([]*devicepb.Device, error) {
+	resp, err := remote.FindDevices(ctx, &devicepb.FindDevicesRequest{
+		IdOrTag: idOrTag,
+	})
+	switch {
+	case err != nil:
+		return nil, trace.Wrap(err)
+	case len(resp.Devices) == 0:
+		return nil, trace.NotFound("device %q not found", idOrTag)
+	case len(resp.Devices) == 1:
+		return resp.Devices[:1], nil
```

Comment:
nit: no reason for slicing here:

```suggestion
		return resp.Devices, nil
```